client/components/section-nav/item.jsx

```diff
@@ -70,7 +70,7 @@ class NavItem extends PureComponent {
 					onClick={ onClick }
 					onMouseEnter={ this.preload }
 					tabIndex={ this.props.tabIndex || 0 }
-					aria-selected={ this.props.selected }
+					aria-current={ this.props.selected }
```

Comment:
This change caused one e2e test to fail, PR https://github.com/Automattic/wp-calypso/pull/34682 fixes it.